docs/essentials/viewport.mdx

```diff
@@ -138,45 +136,57 @@ Update your story through [parameters](../writing-stories/parameters.mdx) to inc
 
 If you need, you can edit them on the shortcuts page.
 
-## API
+## Setting value
 
-### Parameters
+You can set the viewport in the toolbar by clicking on the viewport you want to use.
 
-This addon contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `viewport` namespace:
+If you want to enforce a story to use a viewport, you can do so by setting the `viewport` global in the story:
 
-#### `defaultOrientation`
+```ts filename="Button.stories.ts|tsx" renderer="common" language="ts-4-9"
+// Replace your-framework with the name of your framework
+import type { Meta, StoryObj } from '@storybook/your-framework';
 
-Type: `'portrait' | 'landscape'`
+import { Button } from './Button';
 
-Default: `'portrait'`
+const meta = {
+  component: Button,
+} satisfies Meta<typeof Button>;
 
-Specifies the default orientation used when viewing a story.
+export default meta;
+type Story = StoryObj<typeof meta>;
 
-#### `defaultViewport`
+export const Large: Story = {
+  global: {
+    viewport: { value: 'phone', isRotated: false },
+  },
+};
+```
```

Comment:
@ndelangen if you're ok with it can you adjust this to the following:
````md
## Customizing viewports with globals

By default, the Viewport addon allows you to customize the viewport enabled for a story by selecting from the list of existing viewports in the toolbar. If you need, you can opt into the globals API to configure viewports to ensure that a story always uses a specific viewport regardless of the existing configuration.

To enable it, you'll need to include the [`viewportStoryGlobals`](../api/main-config/main-config-features.mdx#viewportstoryglobals) option in your Storybook configuration (i.e., [`.storybook/main.js|ts`](../configure/index.mdx#configure-your-storybook-project)) and adjust your stories to use this API.

{/* prettier-ignore-start */}

<CodeSnippets path="storybook-addon-viewports-global-api.md" usesCsf3 />

{/* prettier-ignore-end */}

````
Once again, create the snippet file in the mentioned directory (i.e., `_snippets`) and include the following:
````md
```ts filename="Button.stories.ts" renderer="angular" language="ts"
import type { Meta, StoryObj } from '@storybook/angular';

import { Button } from './button.component';

const meta: Meta<Button> = {
  component: Button,
};

export default meta;
type Story = StoryObj<Button>;

export const Large: Story = {
  global: {
    viewport: { value: 'phone', isRotated: false },
  },
};
```

```js filename="Button.stories.js|jsx" renderer="common" language="js"
import { Button } from './Button';

export default {
  component: Button,
};

export const Large = {
  global: {
    viewport: { value: 'phone', isRotated: false },
  },
};
```

```ts filename="Button.stories.ts|tsx" renderer="common" language="ts-4-9"
// Replace your-framework with the name of your framework
import type { Meta, StoryObj } from '@storybook/your-framework';

import { Button } from './Button';

const meta = {
  component: Button,
} satisfies Meta<typeof Button>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Large: Story = {
  global: {
    viewport: { value: 'phone', isRotated: false },
  },
};
```

```ts filename="Button.stories.ts|tsx" renderer="common" language="ts"
// Replace your-framework with the name of your framework
import type { Meta, StoryObj } from '@storybook/your-framework';

import { Button } from './Button';

const meta: Meta<typeof Button> = {
  component: Button,
};

export default meta;
type Story = StoryObj<typeof Button>;

export const Large: Story = {
  global: {
    viewport: { value: 'phone', isRotated: false },
  },
};
```

```js filename="Button.stories.js" renderer="web-components" language="js"
export default {
  component: 'demo-button',
};

export const Large = {
  global: {
    viewport: { value: 'phone', isRotated: false },
  },
};
```

```ts filename="Button.stories.ts" renderer="web-components" language="ts"
import type { Meta, StoryObj } from '@storybook/web-components';

const meta: Meta = {
  component: 'demo-button',
};

export default meta;
type Story = StoryObj;

export const Large: Story = {
  global: {
    viewport: { value: 'phone', isRotated: false },
  },
};
```
````